### PR TITLE
Ufs 250 opportunity finish

### PR DESCRIPTION
**Pull request by ascorbic**
The lambda stuff isn't needed any more, but I've left it in for future reference.

### CODE REVIEW

File: src/components/SetupOpportunity/SetupOpportunity.tsx [hunk 1]
```diff
@@ -10,20 +9,58 @@ import Caption from "@govuk-react/caption";
...
 }
 
-const getAllOpportunities = (opportunity: GetOpportunityQuery) => {
-    const { getOpportunity } = opportunity;
+// const DeleteLink = styled(LinkButton)`
+//     vertical-align: middle;
+//     padding: 0px 10px;
+//     margin-left: 10px;
+//     align-self: flex-start;
+// `;
```
Comment (Darren-Ivey) [inline src/components/SetupOpportunity/SetupOpportunity.tsx]: May as well remove this?

Comment (ascorbic) [reply src/components/SetupOpportunity/SetupOpportunity.tsx]: Done

File: src/components/SetupOpportunity/SetupOpportunity.tsx [hunk 2]
```diff
@@ -53,18 +84,28 @@ export const SetupOpportunity: FC<Props> = ({
...
     useEffect(() => {
-        if (!loading) {
-            setAllOpportunities(getAllOpportunities(opportunity));
+        if (!getOpportunity) {
+            return;
         }
-    }, [loading]);
+
+        setAllWorkflows(getAllWorkflows(getOpportunity));
+    }, [getOpportunity]);
```
Comment (Darren-Ivey) [inline src/components/SetupOpportunity/SetupOpportunity.tsx]: This may result in the flicker effect again.

Comment (ascorbic) [reply src/components/SetupOpportunity/SetupOpportunity.tsx]: Ah, that must have crept in with the merge

Comment (ascorbic) [reply src/components/SetupOpportunity/SetupOpportunity.tsx]: Fixed

**Review by Darren-Ivey — APPROVED** [review]
Looks good.

**Pull request merged**